### PR TITLE
TEP: Token Data Standard

### PR DESCRIPTION
**Pull request by v-lebedev**
A standard interface for tokens (meta)data (in particular [TIP-62](https://github.com/ton-blockchain/TIPs/issues/62) or [TIP-74](https://github.com/ton-blockchain/TIPs/issues/74)).

Moved from [TIP-64](https://github.com/ton-blockchain/TIPs/issues/64).

### CODE REVIEW

**Review by manjaneqx — COMMENTED** [review]
To

**Review by MrG9090 — APPROVED** [review]

**Comment by cumcum8197** [comment]
82d5950caffccd39f985c344fed44c80bb63bb14

File: text/0064-token-data-standard.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - **TEP**: [64](https://github.com/ton-blockchain/TEPs/pull/0)
 - **title**: Token Data Standard
-- **status**: Draft
```
Comment (miladchegeni) [inline text/0064-token-data-standard.md]: Mj

**Comment by cumcum8197** [comment]
82d5950caffccd39f985c344fed44c80bb63bb14

**Comment by cumcum8197** [comment]
82d5950caffccd39f985c344fed44c80bb63bb14

**Comment by cumcum8197** [comment]
82d5950caffccd39f985c344fed44c80bb63bb14

**Comment by cumcum8197** [comment]
82d5950caffccd39f985c344fed44c80bb63bb14

**Review by asim1019 — COMMENTED** [review]
Good

**Pull request merged**